src/explore-education-statistics-admin/src/pages/publication/PublicationContactPage.tsx

```diff
@@ -11,54 +10,29 @@ import SummaryListItem from '@common/components/SummaryListItem';
 import useAsyncHandledRetry from '@common/hooks/useAsyncHandledRetry';
 import useToggle from '@common/hooks/useToggle';
 import React from 'react';
-import { useLocation } from 'react-router';
 
 const PublicationContactPage = () => {
-  const {
-    publicationId,
-    publication: contextPublication,
-    onReload,
-  } = usePublicationContext();
-  const location = useLocation();
-  const lastLocation = useLastLocation();
+  const { publicationId, onReload } = usePublicationContext();
   const [readOnly, toggleReadOnly] = useToggle(true);
 
-  const { value: publication } = useAsyncHandledRetry(
-    async () =>
-      lastLocation && lastLocation !== location
-        ? publicationService.getMyPublication(publicationId)
-        : contextPublication,
+  const { value: contact } = useAsyncHandledRetry(
+    async () => publicationService.getContact(publicationId, true),
     [publicationId],
   );
 
-  const handleSubmit = async ({
-    teamName,
-    teamEmail,
-    contactName,
-    contactTelNo,
-  }: PublicationContactFormValues) => {
-    if (!publication) {
+  const handleSubmit = async (updatedContact: PublicationContactFormValues) => {
+    if (!contact) {
       return;
     }
-    await publicationService.updatePublication(publicationId, {
-      ...publication,
-      contact: {
-        teamName,
-        teamEmail,
-        contactName,
-        contactTelNo,
-      },
-    });
+    await publicationService.updateContact(publicationId, updatedContact);
 
     onReload();
```

Comment:
Don't need this anymore as contact isn't on publication. We actually need to update the `contact` and switch back to the read-only mode during the submit by doing something like this:

```ts
const { value: contact, setValue: setContact } = useAsyncHandledRetry(...);

const handleSubmit = async (updatedContact: PublicationContactFormValues) => {
  ...

  const nextContact = await publicationService.updateContact(publicationId, updatedContact);

  setContact(nextContact);
  toggleReadOnly.off();
};
```

It also seems like we're missing a test case to check that the UI switches back to the read-only state upon successful submit as well.